src/Products/Viewer/Cache/ICustomViewer.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace GroupDocs.Viewer.MVC.Products.Viewer.Cache
+{
+    interface ICustomViewer
+    {
+        GroupDocs.Viewer.Viewer GetViewer();
+        void CreateCache();
+        void Dispose();
```

Comment:
Codacy found an issue: [Either implement 'IDisposable.Dispose', or totally rename this method to prevent confusion.](https://app.codacy.com/manual/GroupDocs/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5933706)